test/optim/test_optim.py

```diff
@@ -793,15 +793,21 @@ def _test_derived_optimizers(self, optimizer_pairs_with_flags, flag):
 
             st_state = state[0]
             mt_state = state[1]
+
+            if reduced_precision:
+                assert_eq_kwargs = {'atol': 1e-5, 'rtol': 1e-4}
+            else:
```

Comment:
can remove the else by moving before control flow

Reply:
yea, that would be good. removing the need for much indentation is usually good.